### PR TITLE
Feature/dwp 2519 s3 directory writer unit test

### PR DESCRIPTION
**Pull request by UdayChokkam**

### CODE REVIEW

File: src/test/kotlin/app/batch/S3DirectoryWriterTest.kt [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    "directory.output=ephemera",
+    "output.batch.size.max.bytes=100000",
+    "source.table.name=ucdata",
+    "compress.output=true",
+    "encrypt.output=true",
+    "aws.region=eu-west-1",
+    "s3.bucket=not_set",
+    "s3.prefix.folder=not_set"
+])
+class S3DirectoryWriterTest {
```
Comment (mark-r-m) [inline src/test/kotlin/app/batch/S3DirectoryWriterTest.kt]: Blank line please

File: src/main/kotlin/app/batch/S3DirectoryWriter.kt [hunk 2]
```diff
@@ -23,6 +24,9 @@ import java.io.ByteArrayInputStream
 class S3DirectoryWriter(keyService: KeyService,
                         cipherService: CipherService) : Writer(keyService, cipherService) {
 
+    @Autowired
+    private  lateinit var s3Client: AmazonS3Client
```
Comment (mark-r-m) [inline src/main/kotlin/app/batch/S3DirectoryWriter.kt]: Double space?

File: src/test/kotlin/app/batch/S3DirectoryWriterTest.kt (hunk 1 above)
Comment (UdayChokkam) [reply src/test/kotlin/app/batch/S3DirectoryWriterTest.kt]: Hi Mark ,  i havent run with integration-all on this branch , will do . Also on the formatting , Do we have any standard formatter used in thhe project which i can  import in intellij editor

Comment (UdayChokkam) [reply src/test/kotlin/app/batch/S3DirectoryWriterTest.kt]: Mark , I have now fixed the suggested formatting and also ran make destroy integration-all which is successful as well

**Pull request merged**